### DISCUSSION TITLE
Pekko-http release in Maven central

### DISCUSSION THREAD

**Discussion by vitalifed** [apache/pekko-http#269] [General]
May I kindly ask to provide the timelines when you are going to release pekko-http to Maven central?

**Comment by pjfanning** [comment]
Hopefully a 1.0.0 release will happen next week. The release discussions appear in https://lists.apache.org/list.html?dev@pekko.apache.org (ASF requirement).

This is the thread about the Pekko HTTP release.

https://lists.apache.org/thread/ogfbr3g2otd35gjct06bj2ll5lbh8bzd
